lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -223,6 +223,10 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
     @Output()
     folderLoaded: EventEmitter<any> = new EventEmitter<any>();
 
+    /** Emitted when the breadcrumb folder title has changed */
+    @Output()
+    breadcrumbFolderTitleEvent = new EventEmitter<string>();
```

Comment:
This event has few wrong things IMO:
1. The name we never call it Event at the end. We try to not use suffix. 
2. The name is not really an indicator of what happened in the component. If the Title change is because we change folder or also for other reason?

Reply:
Thanks for confirming, this has just been pointed out to me. Just about to update to remove the event.